Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -1808,18 +1806,16 @@ public void CreateItem(BaseItem item, BaseItem parent)
         /// <returns>Task.</returns>
         public void CreateItems(IEnumerable<BaseItem> items, BaseItem parent, CancellationToken cancellationToken)
         {
-            var list = items.ToList();
```

Comment:
I think it might be better to keep it here, as down below you're enumerating over this `items` at least twice

Reply:
All callers pass a `List` or `array` atm, so that isn't an issue